metadata.json

```diff
@@ -1,87 +1,67 @@
 {
-  "name": "google-glogging",
-  "version": "0.1.3",
-  "author": "Google",
-  "summary": "A Puppet module to manage Google Stackdriver Logging Agent",
+  "name": "ops-cloud_ops",
+  "version": "0.1.0",
+  "author": "Google Inc.",
+  "summary": "Manage Google Cloud Ops Agent",
   "license": "Apache-2.0",
-  "source": "https://github.com/GoogleCloudPlatform/puppet-google-logging",
-  "project_page": "https://github.com/GoogleCloudPlatform/puppet-google-logging",
-  "issues_url": "https://github.com/GoogleCloudPlatform/puppet-google-logging/issues",
+  "source": "https://github.com/BlueMedora/google-puppet-agents",
+  "project_page": "https://github.com/BlueMedora/google-puppet-agents",
+  "issues_url": "https://github.com/BlueMedora/google-puppet-agents/issues",
+  "dependencies": [
+    { "name": "lwf/remote_file", "version_requirement": ">= 1.1.3"}
+  ],
   "operatingsystem_support": [
     {
-      "operatingsystem": "RedHat",
+      "operatingsystem": "CentOS",
       "operatingsystemrelease": [
-        "6",
-        "7"
+        "7",
+        "8"
       ]
     },
     {
-      "operatingsystem": "CentOS",
+      "operatingsystem": "RedHat",
       "operatingsystemrelease": [
-        "6",
-        "7"
+        "7",
+        "8"
       ]
     },
     {
       "operatingsystem": "Debian",
       "operatingsystemrelease": [
-        "8",
-	"9"
+        "9",
+        "10"
       ]
     },
     {
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
-        "12.04",
-        "14.04",
         "16.04",
-        "16.10",
-	"18.04"
+        "18.04",
+        "20.04"
       ]
     },
     {
       "operatingsystem": "SLES",
       "operatingsystemrelease": [
-        "11-sp4",
-        "12-sp2"
-      ]
-    },
-    {
-      "operatingsystem": "openSUSE",
-      "operatingsystemrelease": [
-        "13"
+        "12",
+        "15"
       ]
     },
     {
-      "operatingsystem": "Windows Server",
+      "operatingsystem": "windows",
       "operatingsystemrelease": [
-        "2008 R2",
-        "2012 R2",
-        "2012 R2 Core",
-        "2016 R2",
-        "2016 R2 Core"
+        "2019",
+        "10"
```

Comment:
This is certainly possible. Configuring the test images for windows will take some time. I would like to add this in a future PR if possible.

Reply:
Yup, that sounds good. We could explore a bit more for whether there is a way to avoid maintaining custom images.

Filed https://github.com/GoogleCloudPlatform/puppet-google-logging/issues/10 for tracking. Resolving this.